src/main/java/com/example/java/maven/gameBlackJack/Card.java

```diff
@@ -0,0 +1,63 @@
+package com.example.java.maven.gameBlackJack;
+
+public class Card {
+    Suit suit;
```

Comment:
private final, and if required add getters

Reply:
Done